internal/api/helpers.go

```diff
@@ -91,3 +91,7 @@ func parseGroupConfig(u *url.URL) (bool, bool, error) {
 
 	return includeGroupConfig, includeGroupConfigOverrides, nil
 }
+
+func int32Ptr(i int32) *int32 {
```

Comment:
You can use `ptr.Int()` from `smithy-go`.

Reply:
Right.